django_multitenant/mixins.py

```diff
@@ -147,7 +146,20 @@ def save(self, *args, **kwargs):
 
     @property
     def tenant_field(self):
-        return self.tenant_id
+        if hasattr(self, "TenantMeta") and "tenant_field_name" in dir(self.TenantMeta):
+            return self.TenantMeta.tenant_field_name
+        if hasattr(self, "TenantMeta") and "tenant_id" in dir(self.TenantMeta):
+            return self.TenantMeta.tenant_id
+        if hasattr(self, "tenant"):
+            raise AttributeError(
+                "Tenant field exists which may cause collusion with tenant_id field. Please rename the tenant field. "
+            )
+        if hasattr(self, "tenant_id"):
+            return self.tenant_id
+
+        raise AttributeError(
```

Comment:
Do this and the other AttributeError create a breaking change in the library, where some consuming code may have worked before but now breaks?

It's OK if it does; I just want to identify it for the changelog and for the next version number.

Reply:
No actually, It will just give a meaningful error and it will be easy to take action. I lost a lot of time just address the issue since there is not such error.